plugins/packages/minio/lib/operations.ts

```diff
@@ -59,3 +59,7 @@ export async function signedUrlForPut(minioClient: MinioClient, queryOptions: ob
 
   return { url };
 }
+
+export async function removeObject(minioClient: MinioClient, queryOptions: object): Promise<object> {
+  return await minioClient.deleteObject(queryOptions['bucket'], queryOptions['objectName']);
```

Comment:
Hey @mslourens the method is [`removeObject`](https://github.com/minio/minio-js/blob/master/examples/remove-object.js)
```suggestion
  return await minioClient.removeObject(queryOptions['bucket'], queryOptions['objectName']);
```

Reply:
Thanks for the review! I implemented your comments.